lib/apiservers/engine/backends/volume.go

```diff
@@ -32,7 +33,7 @@ import (
 const (
 	OptsVolumeStoreKey     string = "VolumeStore"
 	OptsCapacityKey        string = "Capacity"
-	dockerMetadataModelKey string = "DockerMetaData"
+	dockerMetadataModelKey string = "dockerMetaData"
```

Comment:
I was just looking at it and felt that lowercase suited it more. I can change it back :)